### PR TITLE
support bottom-up demo

### PR DESCRIPTION
**Pull request by jin-s13**
#29

### CODE REVIEW

File: demo/bottom_up_video_demo.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        '--kpt-thr', type=float, default=0.3, help='Keypoint score threshold')
+
+    args = parser.parse_args()
+
+    skeleton = [[16, 14], [14, 12], [17, 15], [15, 13], [12, 13], [6, 12],
+                [7, 13], [6, 7], [6, 8], [7, 9], [8, 10], [9, 11], [2, 3],
+                [1, 2], [1, 3], [2, 4], [3, 5], [4, 6], [5, 7]]
+
+    assert args.show or (args.out_video_root != '')
+    assert 'cuda' in args.device
```
Comment (innerlee) [inline demo/bottom_up_video_demo.py:38]: what about the cpu version?

Comment (jin-s13) [reply demo/bottom_up_video_demo.py:38]: We have not tested it on cpu yet. Not sure if it will work..

Comment (jin-s13) [reply demo/bottom_up_video_demo.py]: Support cpu version now.

**Comment by innerlee** [comment]
closing in favor of https://github.com/open-mmlab/mmpose/pull/72

**Pull request closed without merging**